src/Aspire.Cli/Utils/SdkInstallHelper.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using Aspire.Cli.DotNet;
+using Aspire.Cli.Interaction;
+using Aspire.Cli.Resources;
+
+namespace Aspire.Cli.Utils;
+
+/// <summary>
+/// Helper class for managing SDK installation UX and user interaction.
+/// </summary>
+internal static class SdkInstallHelper
+{
+    /// <summary>
+    /// Ensures that the .NET SDK is installed and available, displaying an error message if it's not.
+    /// </summary>
+    /// <param name="sdkInstaller">The SDK installer service.</param>
+    /// <param name="interactionService">The interaction service for user communication.</param>
+    /// <param name="cancellationToken">Cancellation token.</param>
+    /// <returns>True if the SDK is available, false if it's missing.</returns>
+    public static async Task<bool> EnsureSdkInstalledAsync(
+        IDotNetSdkInstaller sdkInstaller,
+        IInteractionService interactionService,
+        CancellationToken cancellationToken = default)
+    {
+        ArgumentNullException.ThrowIfNull(sdkInstaller);
+        ArgumentNullException.ThrowIfNull(interactionService);
+
+        var isSdkAvailable = await sdkInstaller.CheckAsync(cancellationToken);
+
+        if (!isSdkAvailable)
+        {
+            var sdkErrorMessage = string.Format(CultureInfo.InvariantCulture, ErrorStrings.MininumSdkVersionMissing, DotNetSdkInstaller.MinimumSdkVersion);
```

Comment:
@mitchdenny This error message is incorrect when you set overrideMinimumSdkVersion. 

<img width="2574" height="804" alt="image" src="https://github.com/user-attachments/assets/591d2e7b-2531-438c-80e5-45efa16bd0d3" />